RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/card-list.lava

```diff
@@ -19,14 +17,18 @@
             {% assign cciid = item.Id %}
     	    {% assign guid = item.Guid %}
     	    {% assign channelNameParts = item.ContentChannel.Name | Split:' - ' %}
-    	    {% assign type = channelNameParts[1] | Singularize %}
+			{% if item.ContentChannelId == 26 %}
+				{% capture type %}{{ item | Attribute:'Category' }} {{ channelNameParts[1] | Singularize }}{% endcapture %}
+			{% else %}
+    	    	{% assign type = channelNameParts[1] | Singularize %}
+			{% endif %}
     	    {% assign title = item.Title | Replace:"'","’" %}
             {% assign titlesize = "h4" %}
 			{% assign dates = item | Attribute:'Dates','RawValue' %}
 			{% assign dateParts = dates | Split:',' %}
             {% capture subtitle %}
 				{%- comment -%}Check this item's channel to see if dates should be visible{%- endcomment -%}
-				{% if showDate == 'Yes' %}
+				{% if showdates != 'false' and showDate == 'Yes' %}
```

Comment:
Also, `showdates` and `showDate`? That's pretty confusing unless you know what you're looking at. Are they both really necessary? 

Seems like either we wouldn't need 2 variables that are pretty much named the same thing and have to check them both in one conditional OR we could name them things that explain a little more what they are and why they're both needed. I don't think this should hold up the PR, but I do think we should think about how/what we name things.